twitch_rivals/templates/base.html

```diff
@@ -1,15 +1,96 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <link href="{{ url_for('static',filename='style.css') }}" rel="stylesheet" type="text/css"/>
-    <meta charset="UTF-8">
-    <title>Title</title>
-    <div class="navbar">
-            <h1>RivalCast</h1>
-    </div>
+
+    <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content=""/>
+    <meta name="author" content=""/>
+
+    <title>Starter Template for Bootstrap</title>
+
+    <!-- Bootstrap Core CSS -->
+    <link rel="stylesheet" href="https://mdbootstrap.com/wp-content/themes/mdbootstrap4/css/compiled-4.5.14.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/dragula/3.7.2/dragula.min.css">
+    <link rel="stylesheet" type="text/css" href="https://mdbootstrap.com/wp-content/themes/mdbootstrap4/css/mdb-plugins-gathered.min.css">
+    <script src= "https://player.twitch.tv/js/embed/v1.js"></script>
+    <!-- Custom CSS -->
+    <link rel="stylesheet" href="{{ url_for('static', filename='css/base.css') }}">
```

Comment:
can this be static instead of being a jinja template? we know the path ...

Reply:
Solution may be here
https://stackoverflow.com/questions/25034812/use-a-css-stylesheet-on-a-jinja2-template

Reply:
@lkashti sent a StackOverflow answer that shows the current solution, I think we need to stick with the current code, maybe we need to change the design for the pages.